packages/core/admin/ee/server/routes/index.js

```diff
@@ -1,17 +1,14 @@
 'use strict';
 
-// eslint-disable-next-line node/no-extraneous-require
 const { features } = require('@strapi/strapi/lib/utils/ee');
-const featuresRoutes = require('./features-routes');
 
-const getFeaturesRoutes = () => {
-  return Object.entries(featuresRoutes).flatMap(([featureName, featureRoutes]) => {
-    if (features.isEnabled(featureName)) {
-      return featureRoutes;
-    }
+const enableFeatureMiddleware = (featureName) => (ctx, next) => {
+  console.log(featureName, features.isEnabled(featureName));
```

Comment:
i believe this `console.log` was only for debugging purposes ?

Reply:
Yes most likely, Alex added it and I forgot it. I'll remove it right away!